src/services/doppler-api-client.ts

```diff
@@ -205,4 +226,35 @@ export class HttpDopplerApiClient implements DopplerApiClient {
       };
     }
   }
+
+  public async getSubscribers(
+    email: string,
+  ): Promise<ResultWithoutExpectedErrors<SubscriberCollection>> {
+    try {
+      const { jwtToken, userAccount } = this.getDopplerApiConnectionData();
+
+      const { data } = await this.axios.request({
+        method: 'GET',
+        url: `/accounts/${userAccount}/subscribers/`,
```

Comment:
Where do you plan on add the fallback for this method? since it's still not deployed in production.

Reply:
in the component. not in the service because not depends of that